include/slb/type_traits.hpp

```diff
@@ -970,8 +970,22 @@ using std::invoke_result_t;
 // using invoke_result_t = typename slb::invoke_result<F, Args...>::type;
 #endif
 
+// gcc did not implement CWG1558: "Unused arguments in alias template
+// specializations" until version 5.
+#if __GNUC__ < 5
```

Comment:
If we're not using GCC, `__GNUC__` would evaluate to `0` and we would use this code, right?

Reply:
Yes, good catch.